gpflow/utilities/utilities.py

```diff
@@ -420,6 +420,11 @@ def traverse_module(
             new_acc = (f"{path}['{term_idx}']", new_state)
             new_state = traverse_module(subterm, new_acc, update_cb, target_types)
     elif isinstance(m, tf.Module):
+        items = vars(m)
+        if isinstance(m, tfp.bijectors.Bijector) and "_parameters" in items:
```

Comment:
`_parameters` was added in tensorflow_probability 0.10 release.

Reply:
https://github.com/tensorflow/probability/commit/a95c160abc2c8ab9f8502817a9cea7062a8103e8